### PR TITLE
Fixed action workflow error in stale-issue.yaml

### PR DESCRIPTION
**Pull request by shmishra99**
**Issue**: Workflow is giving error while running.Error msg-The workflow is not valid. .github/workflows/stale-issues.yml 
**Fixed**: Followed stale/action workflow [docs](https://github.com/actions/stale#any-of-labels).
Changed any-of-labels value to ',' string.
Changed only-issue-labels to only-labels to work for both PR and issues.

### CODE REVIEW

**Review by mihaimaruseac — APPROVED** [review]

**Review by ymodak — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Comment by gbaned** [comment]
@shmishra99 Can you please resolve conflicts? Thank you!

File: .github/workflows/stale-issues.yml [hunk 1]
```diff
@@ -43,17 +43,18 @@ jobs:
...
-          stale-pr-message: "This PR is stale because it has been open for 28 days with no activity."
+          stale-pr-message: "This PR is stale because it has been open for 14 days with no activity.It will be closed if no further activity occurs. Thank you."
+          close-pr-message: "This PR was closed because it has been inactive for 14 days since being marked as stale.Please reopen if you'd like to work on this further."
           repo-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Contribution issues
         uses: actions/stale@v5
         with:
           days-before-issue-stale: 180
-          days-before-issue-close: 365
+          days-before-issue-close: 7
```
Comment (mihaimaruseac) [inline .github/workflows/stale-issues.yml:53]: Why did you change this?
```suggestion
          days-before-issue-close: 365
```

**Review by mihaimaruseac — APPROVED** [review]

File: .github/workflows/stale-issues.yml [hunk 2]
```diff
@@ -17,7 +17,6 @@ name: Close inactive issues
 on:
   schedule:
     - cron: "30 1 * * *"
```
Comment (mihaimaruseac) [inline .github/workflows/stale-issues.yml:19]:
```suggestion
    - cron: "30 1 * * *"

```

Keep that empty line, it does not make sense to remove

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

File: .github/workflows/stale-issues.yml [hunk 3]
```diff
@@ -43,17 +44,18 @@ jobs:
             Please reopen if you'd like to work on this further.
           days-before-pr-stale: 14
           days-before-pr-close: 14
-          stale-pr-message: "This PR is stale because it has been open for 28 days with no activity."
+          stale-pr-message: "This PR is stale because it has been open for 14 days with no activity.It will be closed if no further activity occurs. Thank you."
```
Comment (bhack) [inline .github/workflows/stale-issues.yml:47]: Is it ok to have `.` without a space or is it a small typo?

File: .github/workflows/stale-issues.yml [hunk 4]
```diff
@@ -43,17 +44,18 @@ jobs:
...
+          close-pr-message: "This PR was closed because it has been inactive for 14 days since being marked as stale.Please reopen if you'd like to work on this further."
```
Comment (bhack) [inline .github/workflows/stale-issues.yml:48]: Ditto

File: .github/workflows/stale-issues.yml (hunk 3 above)
Comment (mihaimaruseac) [inline .github/workflows/stale-issues.yml:47]:
```suggestion
          stale-pr-message: "This PR is stale because it has been open for 14 days with no activity. It will be closed if no further activity occurs. Thank you."
```

File: .github/workflows/stale-issues.yml (hunk 4 above)
Comment (mihaimaruseac) [inline .github/workflows/stale-issues.yml:48]:
```suggestion
          close-pr-message: "This PR was closed because it has been inactive for 14 days since being marked as stale. Please reopen if you'd like to work on this further."
```

**Review by mihaimaruseac — APPROVED** [review]

**Review by mihaimaruseac — APPROVED** [review]
